### PR TITLE
Fixed Inconsistent Sidebar Styles #1

### PR DESCRIPTION
**Pull request by a-n-g-i-e-r-i**
fixes [#1](https://github.com/sf-wdi-25/publify_debugging_lab/issues/1)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,40 +1,5 @@
 source 'https://rubygems.org'
-
```
Comment (tgaff) [inline Gemfile:2]: @isangieri Why are you changing our gem file in this commit?  Is that related to the bug?

File: config/database.yml [hunk 2]
```diff
@@ -9,7 +7,7 @@ connection: &connection
 
 
 development:
-  database: publify_dev
```
Comment (tgaff) [inline config/database.yml:12]: I'm confused.  What do these database changes have to do with the fix?

File: app/assets/stylesheets/user-styles.css [hunk 3]
```diff
@@ -35,3 +35,7 @@ img.right {
 #sidebar .sidebar-body li {
   list-style-type: circle;
 }
+
+#archives li {
```
Comment (nathanallen) [inline app/assets/stylesheets/user-styles.css]: Please defend addition of redundant style. See lines 35-37, above.
